packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -218,7 +218,7 @@ export async function load_node({
 
 				if (response) {
 					const proxy = new Proxy(response, {
-						get(response, key, receiver) {
+						get(response, key, _receiver) {
```

Comment:
Why not just remove this one?

Reply:
Type of `ProxyHandler#get` is `get?(target: T, p: string | symbol, receiver: any): any;`.
This is defined by ES2015.
Personally, in this case, I prefer to notate args properly. (In this case, actually, there are 3 args, so I notate 3 args.)
(Maybe because I like typed languages.)

But this is just personal preference.
Therefore if you like to remove `_receiver`, of cause I will remove it.